sdk/core/azure-core/src/main/java/com/azure/android/core/util/Configuration.java

```diff
@@ -0,0 +1,318 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
We probably need to get together with Anu and determine how to re-implement this. There's no way to set system properties or environment variables on Android so that doesn't apply.

The platform native way would be via [SharedPreferences](https://developer.android.com/reference/android/content/SharedPreferences) which are accessible via the `Context`. However we don't want to couple this directly to the Context itself, and as we discussed a bit previously we should access any platform services or anything contained within the Context by way of an adapter interface or provider. That will allow us to write unit tests for this without having any dependencies on the Emulator.